python/mxnet/numpy/linalg.py

```diff
@@ -67,6 +70,53 @@ def matrix_rank(M, tol=None, hermitian=False):
     return _mx_nd_np.linalg.matrix_rank(M, tol, hermitian)
 
 
+def vecdot(a: ndarray, b, ndarray, axis: Optional[int] =None) -> ndarray:
```

Comment:
```def vecdot(a: ndarray, b, ndarray, axis: Optional[int] =None) -> ndarray:```
How about adding the same function style in the entire file by giving the type-name explicitly?